### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by Veykril**
Closes https://github.com/rust-lang/rust-analyzer/issues/12944

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Before submitting, please make sure that you're not running into one of these kn
 Otherwise please try to provide information which will help us to fix the issue faster. Minimal reproducible examples with few dependencies are especially lovely <3.
 -->
 
-**rust-analyzer version**: (eg. output of "rust-analyzer: Show RA Version" command)
+**rust-analyzer version**: (eg. output of "rust-analyzer: Show RA Version" command, accessible in VSCode via <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>)
```
Comment (lnicola) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: Does that work on a Mac?  Would <kbd>F1</kbd> be easier?

Comment (Veykril) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: This key-combo is what the docs state https://code.visualstudio.com/api/references/contribution-points#contributes.commands
> By default, commands show in the Command Palette (Ctrl+Shift+P) but they can also show in other menus.

Comment (lnicola) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: It's Cmd+Shift+P on Macs: https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf.

**Pull request merged**